basic/code26.s

```diff
@@ -1,12 +1,6 @@
 ;most references to kernal are defined here
 ;
-erexit	cmp #$f0        ;check for special case
```

Comment:
FYI, this is certainly orphan code.  This logic only applied on C64 when opening or closing an RS-232 device. It appears that the C64 ROM created a buffer at the top of BASIC RAM and this was the mechanism used to inform BASIC of the change.  It would never be true on X16 since that code has already been removed.